Makefile

```diff
@@ -58,6 +58,11 @@ pkg/rest/zz_docs: $(wildcard pkg/rest/*.go pkg/korrel8r/*.go) $(SWAG)
 lint: $(VERSION_TXT) $(GOLANGCI_LINT) ## Run the linter to find and fix code style problems.
 	$(GOLANGCI_LINT) run --fix
 
+.PHONY: fmt
+fmt: $(SHFMT) ## Run go fmt against code.
+	go fmt ./...
+	$(SHFMT) -l -w ./**/*.sh
```

Comment:
Suggest running $(SHFMT) as  part of the lint: target. That will simplify the git actions and lets devs ensure they are "lint and format clean" with one command.